install-project.sh

```diff
@@ -1,16 +1,23 @@
-
 slug=$1
-MVNOPTIONS=$2
+MVNOPTIONS="$2 -Dmaven.repo.local=$AZ_BATCH_TASK_WORKING_DIR/input/dependencies"
 USER=$3
 module=$4
 sha=$5
 dir=$6
 fullTestName=$7
 RESULTSDIR=$8
+projectname=${slug%/*}
 
 modifiedslug=$(echo ${slug} | sed 's;/;.;' | tr '[:upper:]' '[:lower:]')
 short_sha=${sha:0:7}
 modifiedslug_with_sha="${modifiedslug}-${short_sha}"
+modified_module=$(echo ${module} | cut -d'.' -f2- | cut -c 2- | sed 's/\//+/g')
+
+if [[ ! -f "$AZ_BATCH_TASK_WORKING_DIR/input/"${modifiedslug_with_sha}=${modified_module}".zip" ]]; then
```

Comment:
Should we delete this if clause? This clause seems to prevent any project from actually compiling if it does not already exist in the input container. 

This script is also missing an if clause to prevent projects from being installed if the project is already in the input container.

Reply:
Ops, I put the "!" by mistake in the if statement, I am fixing